x/gov/src/types/proposal.rs

```diff
@@ -0,0 +1,62 @@
+use chrono::{DateTime, SubsecRound, Utc};
+use gears::types::base::coin::Coin;
+use serde::{Deserialize, Serialize};
+
+use crate::keeper::KEY_PROPOSAL_PREFIX;
+
+// Slight modification of the RFC3339Nano but it right pads all zeros and drops the time zone info
+const SORTABLE_DATE_TIME_FORMAT: &str = "%Y-%m-%dT&H:%M:%S.000000000"; // TODO:NOW Is %S for seconds?
+
+const KEY_ACTIVE_PROPOSAL_QUEUE_PREFIX: [u8; 1] = [0x01];
+const KEY_INACTIVE_PROPOSAL_QUEUE_PREFIX: [u8; 1] = [0x02];
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct Proposal {
+    pub proposal_id: u64,
+    pub content: Vec<u8>, // TODO
+    pub status: ProposalStatus,
+    pub final_tally_result: (), // TODO: https://github.com/cosmos/cosmos-sdk/blob/d3f09c222243bb3da3464969f0366330dcb977a8/x/gov/types/gov.pb.go#L289
+    pub submit_time: (),
+    pub deposit_end_time: DateTime<Utc>,
+    pub total_deposit: Vec<Coin>,
```

Comment:
Are there any additional constraints on the list of coins? Do we need a new type?

Reply:
For now, it's okay, but later maybe I add a new type or use `SendCoins`( it needs to be renamed)

Reply:
Fair enough. I agree with the renaming, `Vec<Coin>` seems to be appearing more and more.